### ISSUE TITLE
v1.1.0 seems to be trying to use incorrect static files

### ISSUE THREAD

**Issue by dkarchmer** [jmrivas86/django-json-widget#51]
* django-json-widget version: v1.1.0
* Django version: 3.1
* Python version: 3.7
* Operating System: Linux

### Description

Just upgrade to Django v3.1 and with it, to django-json-version 1.1.0. After I run `collectstatic`, I get the following two files under my `statics` directory

- `dist/jsoneditor.min.css`
- `dist/jsoneditor.min.js`

which seems to be the same as version v1.0.1.

The problem is that when running the server, the page fails to load. Looking at the browser console, I can see it is erroring out trying to find `dist/jsoneditor.css` and `dist/jsoneditor.js` (see how it is not the `.min.` version)

![image](https://user-images.githubusercontent.com/3330496/107111286-ad807680-6803-11eb-9c54-a6cc25f32f35.png)

### What I Did

I didn't do anything different, so unless something changed in the way Django 3.1 works, this may be a problem with the new v1.1.0 version

**Comment by dkarchmer** [comment]
Seems like the bug was introduced here:

https://github.com/jmrivas86/django-json-widget/commit/1e7b0470727a567aa60d1459853423bdb4d6c75c#diff-c92ce623c95bf852173e45c7aa3fdbecb97463b16a1a9a6834a2d9edd87e5a8dR11

See how the defaults are no longer `.min` files

**Comment by dkarchmer** [comment]
Would be nice to add the optional (not so optional with this bug) settings to the README:

```
JSON_EDITOR_JS = 'https://cdnjs.cloudflare.com/ajax/libs/jsoneditor/9.1.9/jsoneditor.min.js'
JSON_EDITOR_CSS = 'https://cdnjs.cloudflare.com/ajax/libs/jsoneditor/9.1.9/jsoneditor.min.css'
```

**Comment by pmtarantino** [comment]
Hello. I found the same problem. Thanks for reporting it @dkarchmer. I revert it back to version 1.0.1.

**Comment by illing2005** [comment]
We're having the same issue

**Comment by ash9431** [comment]
same issue 😢

**Comment by StevenMapes** [comment]
This is fixed in version 1.1.1 - https://github.com/jmrivas86/django-json-widget/releases/tag/v1.1.1

**Comment by jmrivas86** [comment]
Fixed in version 1.1.1 by @StevenMapes 👏  in https://github.com/jmrivas86/django-json-widget/pull/52
